tslint.json

```diff
@@ -1,6 +1,8 @@
 {
     "extends": ["tslint:latest"],
     "rules": {
-        "no-implicit-dependencies": false
+        "no-implicit-dependencies": false,
```

Comment:
why do we want implicit dependencies again?

Reply:
Hound can't handle it at the moment. See: https://github.com/houndci/hound/issues/1522

Reply:
Got it